### PR TITLE
Add environment details in PR body

### PR DESCRIPTION
**Pull request by shreekarSS**
Signed-off-by: Shreekara <sshreeka@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/kebechet/issues/1099
…

### CODE REVIEW

**Comment by KPostOffice** [comment]
What do you think about adding the env details to the body in this function and removing the code in other bodies? This way all PRs will have the env info.

https://github.com/thoth-station/kebechet/blob/c864a6babfa2727392dd62b56915739cc3c5b051/kebechet/managers/manager.py#L164

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -400,7 +399,7 @@ def _open_merge_request_update(
         )
 
         _LOGGER.info("Creating a new pull request to update dependencies.")
-        merge_request = self.project.create_pr(
```
Comment (KPostOffice) [inline kebechet/managers/update/update.py:403]: There is another call in managers/version/version.py

**Review by KPostOffice — COMMENTED** [review]
This looks good. What do you think about extending this for issues created by as well?

File: kebechet/managers/update/update.py (hunk 1 above)
Comment (shreekarSS) [reply kebechet/managers/update/update.py:403]: ack

File: kebechet/managers/manager.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def create_pr(self, title: str, body: str, source_branch: str, target_branch: st
         """Create a PR but defaults to opening PR within a fork rather than on parent fork."""
         return self.project.create_pr(
             title=title,
-            body=body,
+            body=body + "\n" + str(self.get_environment_details()),
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply kebechet/managers/manager.py]: Ack

**Review by KPostOffice — APPROVED** [review]

**Comment by goern** [comment]
@shreekarSS has this feature been released and deployed?

**Pull request merged**